machineconfiguration/v1/0000_80_controllerconfig.crd.yaml

```diff
@@ -49,7 +49,6 @@ spec:
               required:
                 - additionalTrustBundle
                 - baseOSContainerImage
-                - baseOSExtensionsContainerImage
```

Comment:
Removing options from the enum, why has that change come into this PR? Was the CRD out of date in MCO?

Reply:
These specifically in ControllerConfig I did adjust. 

So last time ControllerConfig got regenerated it was a one-off by a "3rd party" PR and those fields got set wrong...and we were apparently in a hurry.  It effectively didn't matter to the MCO because it always supplies the whole structure but we marked it as "probably not great, we should fix it when we migrate the API":  https://github.com/openshift/machine-config-operator/pull/3567/files#r1114291604

And osImageURL is optional now that we're moved to native container format images and that just never got adjusted (again, I think we're supplying the whole thing every time so I don't think it actually matters, but it was wrong).

Reply:
Right, so in this case, you made a breaking change in the 4.13 branch, this is undoing that breaking change, I'm ok with that